RELEASES.md

```diff
@@ -90,13 +107,21 @@ Unreleased.
   These methods do not affect the size of the pre-allocated pool.
   [#6835](https://github.com/bytecodealliance/wasmtime/pull/6835)
 
-* Builder methods for WASI contexts onw use `&mut self` instead of `self`.
+* Builder methods for WASI contexts now use `&mut self` instead of `self`.
   [#6770](https://github.com/bytecodealliance/wasmtime/pull/6770)
 
 * Native unwinding information is now properly disabled when it is configured to
   be turned off.
   [#6547](https://github.com/bytecodealliance/wasmtime/pull/6547)
 
+* Wasmtime's minimum supported Rust version (MSRV) is now 1.70.0. Wasmtime's
+  MSRV policy is now additionally documented.
```

Comment:
Can we briefly say what it is here, for folks skimming? (Something simple like "now additionally documented: we will support two releases behind the latest stable (N-2). See the PR discussion for more details.")